src/main/java/az/edu/strangers/Fish.java

```diff
@@ -0,0 +1,13 @@
+package az.edu.strangers;
+
+public class Fish extends Pet {
+
+    public Fish(String nickName, Integer age, Integer trickLevel, String[] habits) {
+        super(Species.BIRD, nickName, age, trickLevel, habits);
```

Comment:
Species.BIRD shouldn't be in super's constructor. It should be in the class's constructor